packages/core/schema/lavamoat-policy.v0-0-1.schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "LavaMoat Policy Schema",
+  "$id": "lavamoat-policy.v0-0-1",
+  "type": "object",
+  "description": "Schema for LavaMoat policy files",
+  "properties": {
+    "resources": {
+      "description": "Describe the resources available to your application and direct dependencies",
+      "title": "Resources",
+      "type": "object",
+      "$comment": "This one or more valid npm package names delimited by a \">\" character",
+      "patternProperties": {
+        "^(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*(>(@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*)*$": {
```

Comment:
(should) Let's make sure we don't break on actual packages. https://github.com/npm/validate-npm-package-name mentions legacy names too. This is scary :) 
There values are created by the aa package always, so how about we define the pattern to be more trusting? Something like:
`^[^>]+(>[^>]+)*$`

There's no point validating the shape of this string by the schema, because it gets a much thorough validation against actual node_modules elsewhere.

Reply:
Fair enough